src/commands/relay.mjs

```diff
@@ -69,12 +72,35 @@ export class RelayCommand extends BaseCommand {
       throw new MissingArgumentError('Node IDs must be specified')
     }
 
+    const networkJsonString = this.prepareNetworkJsonString(nodeIDs)
+    valuesArg += ` --set config.HEDERA_NETWORK='${networkJsonString}'`
+    return valuesArg
+  }
+
+  // created a json string to represent the map between the node keys and their ids
+  // output example '{"node-1": "0.0.3", "node-2": "0.004"}'
+  prepareNetworkJsonString (nodeIDs = []) {
+    if (!nodeIDs) {
+      throw new MissingArgumentError('Node IDs must be specified')
+    }
+
+    let jsonString = '{'
```

Comment:
Instead of the creating a json string, create an object first and then just use `JSON.stringify`.

Reply:
Changed to using JSON.stringify. Thanks